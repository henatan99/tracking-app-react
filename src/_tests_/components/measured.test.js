import React from 'react';
import Enzyme from 'enzyme';
import { shallow, mount } from 'enzyme';
import { create } from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Adapter from 'enzyme-adapter-react-16';

import Measured from '../../components/measured';

Enzyme.configure({ adapter: new Adapter() });


// it('renders when no props passed', () => {
//   const tree = create(<Measured />).toJSON;
//   expect(tree).toMatchSnapshot();
// });

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Measured />);
});

describe('Measured', () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
  // it('should render one <h1>', () => {
  //   const wrapper = mount(
  //     <MemoryRouter>
  //       <Measured auth={{ loading: false }} />
  //     </MemoryRouter>
  //   );
  //   expect(wrapper.find('div').length).toBe(12);
  // });
});
    
