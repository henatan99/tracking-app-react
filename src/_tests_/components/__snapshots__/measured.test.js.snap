// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Measured renders correctly 1`] = `ShallowWrapper {}`;

exports[`renders when no props passed 1`] = `[Function]`;
